chart/templates/server-ide-configmap.yaml

```diff
@@ -53,13 +53,13 @@ options:
     orderKey: "00"
     title: "VS Code"
     type: "browser"
-    logo: "vscode"
+    logo: "/ide/image/ide-logo/vscode.svg"
```

Comment:
How is it resolved to an absolute URL?

In gitpod.io it should be resolved relative to `ide.gitpod.io`, but in self-hosted against `gitpod.io/ide`.